servers/physics_2d/collision_solver_2d_sw.cpp

```diff
@@ -35,7 +35,7 @@
 //#define collision_solver gjk_epa_calculate_penetration
 
 bool CollisionSolver2DSW::solve_static_line(const Shape2DSW *p_shape_A, const Transform2D &p_transform_A, const Shape2DSW *p_shape_B, const Transform2D &p_transform_B, CallbackResult p_result_callback, void *p_userdata, bool p_swap_result) {
-	const LineShape2DSW *line = static_cast<const LineShape2DSW *>(p_shape_A);
+	const WorldMarginShape2DSW *line = static_cast<const WorldMarginShape2DSW *>(p_shape_A);
 	if (p_shape_B->get_type() == PhysicsServer2D::SHAPE_LINE) {
```

Comment:
In order to keep things consistent in the physics server, we should also rename these:
-`SHAPE_LINE` to `SHAPE_WORLD_MARGIN`
-`solve_static_line()` to `solve_static_world_margin()`
-`line` local variable to `world_margin`